pttrack/models.py

```diff
@@ -194,29 +201,29 @@ class Patient(Person):
 
     outcome = models.ForeignKey(Outcome, null=True, blank=True)
 
-    address = models.CharField(max_length=200)
+    address = models.CharField(max_length=200, verbose_name=_('address'))
 
     city = models.CharField(max_length=50,
-                            default="St. Louis")
+                            default="St. Louis", verbose_name=_('city'))
     state = models.CharField(max_length=2,
-                             default="MO")
+                             default="MO", verbose_name=_('state'))
     zip_code = models.CharField(max_length=5,
-                                validators=[validators.validate_zip])
+                                validators=[validators.validate_zip], verbose_name=_("Zip Code"))
     country = models.CharField(max_length=100,
-                               default="USA")
+                               default=_("USA"), verbose_name=_('country'))
 
 
     pcp_preferred_zip = models.CharField(max_length=5,
                                          validators=[validators.validate_zip],
                                          blank=True,
-                                         null=True)
+                                         null=True, verbose_name=_("pcp preferred zip"))
```

Comment:
Does this create the need for a database migration? What happens when you run `manage.py makemigrations`?